clang/include/clang/Format/Format.h

```diff
@@ -922,8 +922,39 @@ struct FormatStyle {
     ///   };
     ///   int f();
     ///   int f() { return 1; }
+    ///   int LongName::
+    ///       AnotherLongName();
     /// \endcode
```

Comment:
Change the comments (lines 917-927) to something like the following:
```
    /// This is **deprecated**. See ``Automatic`` below.
```

Reply:
Done